### PR TITLE
Updated Typos and added Details for AVM FRITZ!Box 7590

### PR DESCRIPTION
**Pull request by Wlanfr3ak**
Added Comments Link, fixing Typos and correct ISDN Port Name and Airflow added.
Original File: https://github.com/netbox-community/devicetype-library/blob/master/device-types/AVM/7590.yaml

### CODE REVIEW

File: device-types/AVM/7590.yaml [hunk 1]
```diff
@@ -1,10 +1,12 @@
 ---
 manufacturer: AVM
-model: FRITZ!box 7590
-slug: frtizbox-7590
-part_number: 2000 2804
+model: FRITZ!Box 7590
+slug: fritzbox-7590
+part_number: '20002784'
```
Comment (danner26) [inline device-types/AVM/7590.yaml:5]: `part_number` does not need quotes around it, please remove that

**Review by danner26 — CHANGES_REQUESTED** [review]

**Review by Wlanfr3ak — COMMENTED** [review]
Change like the request in pr

**Comment by danner26** [comment]
I am sorry, I lied to you. It interprets it as a non-string for whatever reason. Please change it back and it should be fine @Wlanfr3ak

**Review by Wlanfr3ak — COMMENTED** [review]
Fix it again (syntax highlighting give me the hint before ;-)

**Comment by danner26** [comment]
> Fix it again (syntax highlighting give me the hint before ;-)

Yeah my bad, its been a long day lol

**Review by danner26 — APPROVED** [review]

**Pull request merged**